spec/hbw/features/tasks/task_list_with_filter_bp_spec.rb

```diff
@@ -0,0 +1,68 @@
+feature 'Check table with tasks (filter_bp is true)', js: true do
+  let(:first_task_due_date)  { DateTime.parse('2016-07-30T12:07:59').in_time_zone }
+  let(:second_task_due_date) { DateTime.parse('2017-07-30T12:07:59').in_time_zone }
+
+  before(:each) do
+    set_camunda_api_mock_file('spec/hbw/features/tasks/task_list_with_filter_bp_mock.yml')
+
+    set_bp_filter(true)
+
+    order_type = FactoryBot.create(:order_type, :support_request)
+    FactoryBot.create(:order, order_type: order_type) # ORD-1
+
+    user = FactoryBot.create(:user)
+
+    signin(user.email, user.password)
+    expect(page).not_to have_content 'Sign in'
+    expect(page).to     have_content 'Orders list'
+
+    click_on 'Tasks'
+    expect_widget_presence
+    expect(page).to have_content 'Open tasks'
+    expect(page).to have_content 'My tasks (2)'
+    expect(page).to have_content 'Unclaimed tasks (3)'
+  end
+
+  after(:each) do
+    set_bp_filter(false)
+  end
+
+  describe 'task list rendered' do
+    it 'for tasks assigned to current user' do
+      expect(tasks_table_header).to eq ['Priority', 'Title', 'Task type', 'Task description', 'Created/Due']
+
+      expect(tasks_table_content).to eq(
+        [
+          ['Medium', 'Assigned task', ' Test name', '—', "expired (#{years_since(first_task_due_date)}y past due date)"],
+          ['High', 'Other assigned task', ' Test name', 'Some test description', "expired (#{years_since(second_task_due_date)}y past due date)"]
+        ]
+      )
+    end
+
+    it 'for unclaimed tasks' do
+      click_on_tab 'Unclaimed tasks (3)'
+
+      expect(tasks_table_header).to eq ['Priority', 'Title', 'Task type', 'Task description', 'Created/Due', '']
+
+      expect(tasks_table_content).to eq(
+        [
+          ['Medium', 'Unassigned task', ' Test name', '—', "expired (#{years_since(first_task_due_date)}y past due date)", 'Claim'],
+          ['High', 'Other unassigned task', ' Test name', 'Some test description', "expired (#{years_since(second_task_due_date)}y past due date)", 'Claim'],
+          ['High', 'Check test form', ' Test name', '—', '30 Jun 2016', 'Claim']
+        ]
+      )
+    end
+  end
```

Comment:
You just respond with tasks of the same process definition and render it, so what exactly did you want to test using two lists of tasks?

Reply:
Ok, let's leave only assigned tasks